Source/Controllers/DataControllers/SPTableData.m

```diff
@@ -464,10 +466,19 @@ - (NSDictionary *) informationForTable:(NSString *)tableName fromDatabase:(NSStr
 
     // Retrieve the CREATE TABLE syntax for the table
     SPMySQLResult *theResult;
-    if (database)
-        theResult = [mySQLConnection queryString:[NSString stringWithFormat:@"SHOW CREATE TABLE %@.%@", [database backtickQuotedString], [tableName backtickQuotedString]]];
-    else
-        theResult = [mySQLConnection queryString:[NSString stringWithFormat:@"SHOW CREATE TABLE %@", [tableName backtickQuotedString]]];
+    NSString *queryStr;
+
+    if (database){
+        queryStr = [NSString stringWithFormat:@"SHOW CREATE TABLE %@.%@", [database backtickQuotedString], [tableName backtickQuotedString]];
+        theResult = [mySQLConnection queryString:queryStr];
```

Comment:
Could move theResult to outside/after the IF statement as it's the same for both the IF and ELSE conditions